### ISSUE TITLE
Dragging and resizing objects in RTL language based websites

### ISSUE THREAD

**Issue by aneeshtan** [GoogleForCreators/web-stories-wp#3324]
Hi, 

I just noticed that there is a major bug in dragging and resizing objects in RTL languages. While I try to move the object it would fly to opposite side of the layout. 

Also we weren't able to publish any story as it says: 

Failed to save the story

**Comment by LuckynaSan** [comment]
@aneeshtan Thank you for reporting this issue. Your first issue may be related to GitHub issue #508 which we are investigating. Regarding the second issue, if you wish to share a video of your experience with publishing the story that will be great. You can use something like the app [recordit](https://recordit.co/) to make a quick gif/video and share the link.

**Comment by swissspidy** [comment]
The first issue is indeed being tracked in #508.

As for the "Failed to save the story" issue, please also check your browser's developer tools (press <kbd>F12</kbd>) if you see any errors. Next, check the "Network" tab to debug any outgoing requests. It should look a bit like this:

![image](https://user-images.githubusercontent.com/68001147/86907869-51e9ef80-c133-11ea-8b80-61c3af8181d5.png)

If you click on that item and then the "Response" tab, what do you see? Can you share a screenshot?

**Comment by LuckynaSan** [comment]
@aneeshtan I want to follow up on your issue regarding "Failed to save the story" error. 

- Are you still experiencing this? 
- Can you check the Network tab in your browser developer tool and provide a screenshot of the "Response" tab?

**Comment by aneeshtan** [comment]
Hi, it was related to the firewall setting on Cloudflare that blocked any url such as wp-json. I fixed that issue by whitelisting that URL on Cloudflare. But the bug with RTL languages with dragging objects is preventing us from using this plugin.

**Comment by LuckynaSan** [comment]
@aneeshtan Thanks for the information! I will close this issue out because you can follow progress on the bug with RTL languages in GitHub issue 508 mentioned above. If you continue to experience any other issues, please feel free to post a topic on the [Web Stories support forum](https://wordpress.org/support/plugin/web-stories/) in WordPress.org.
